React/README.md

```diff
@@ -1 +1,26 @@
 # React components
+
+## Modal Component:
```

Comment:
This README should be inside the component, it acts as a documentation for the component. You have put it in the wrong place. Also use the README template in the repository to make a README for your component.